### PR TITLE
[Model Runner V2] Enable GraniteMOE for MRv2 by default

### PR DESCRIPTION
**Pull request by yewentao256**
## Purpose

Part of https://github.com/vllm-project/vllm/issues/41286

We enable GraniteMOE as a next step because their are plenty of tests for it

## Test

Covered in CI

### CODE REVIEW

**Review by njhill — APPROVED** [review]

**Review by yewentao256 — COMMENTED** [review]
[buildkite/ci/pr/attention-benchmarks-smoke-test-b200](https://buildkite.com/vllm/ci/builds/71850#019ec024-8025-432f-b78d-e6e47759c8fa) has been fixed in main

[buildkite/ci/pr/amd-v1-sample-plus-logits-mi325-1](https://buildkite.com/vllm/ci/builds/71850#019ec024-8408-4627-96b3-8ad33c2dc53e) is a known issue, let's remove the run all test label now.
